src/explore.ts

```diff
@@ -207,6 +208,7 @@ function computeFileSizes(
   const sourceMapCommentBytes = Buffer.byteLength(sourceMapComment);
   const eolBytes = getOccurrencesCount(eol, fileContent) * Buffer.byteLength(eol);
   const totalBytes = Buffer.byteLength(fileContent);
+  files[EOL_KEY] = eolBytes;
```

Comment:
Please move it to `exploreBundle` where other special files are added.
Or move `if (!options.excludeSourceMapComment)` and `if (!options.onlyMapped)` here